tools/docs/config.ts

```diff
@@ -233,10 +234,29 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
         `\n${option.description}\n\n` +
         genTable(Object.entries(el), option.type, option.default);
 
+      if (el.advancedUse) {
+        if (
+          option.name === 'excludeCommitPaths' ||
+          option.name === 'group' ||
+          option.name === 'followTag'
+        ) {
+          configOptionsRaw[headerIndex] += generateAdvancedUse();
```

Comment:
i think we should always add it after the header.

Reply:
in "commitMessages..." the warning text appears at the end right now, we can move it after the table.

Reply:
should at least always at same place and very early